src/test/java/com/vmware/gemfire/testcontainers/SimpleSecurityManager.java

```diff
@@ -61,7 +61,7 @@ public Object authenticate(final Properties credentials) throws AuthenticationFa
     }
     String username = credentials.getProperty(USER_NAME);
     String password = credentials.getProperty(PASSWORD);
```

Comment:
why do you not just have defaults for the getter? Then you never have to test of `!= null` or `isEmpty()`..

Reply:
If no username is provided (i.e. `username == null`), then authentication should fail.